style.css

```diff
@@ -35,4 +35,1131 @@
     src: local('Montserrat Bold'), local('Montserrat-Bold'),
          url('../fonts/montserrat-v14-latin-700.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
          url('../fonts/montserrat-v14-latin-700.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
-  }
\ No newline at end of file
+  }
+
+
+
+html {
+    box-sizing: border-box;
+}
+*, *::before, *::after {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=6044979)